executor/test_linux.h

```diff
@@ -43,7 +43,7 @@ static int test_one(int text_type, const char* text, int text_size, int flags, u
 		printf("cpu mmap failed (%d)\n", errno);
 		return 1;
 	}
-	int vm_mem_size = 96 << 10;
+	int vm_mem_size = 24 * SYZ_PAGE_SIZE; // Allocate what syzkaller normally allocates
```

Comment:
"what syzkaller normally allocates" would be `SYZ_NUM_PAGES * SYZ_PAGE_SIZE`.

Also we generally don't refer to syzkaller in the third person in syzkaller. This _is_ syzkaller as well. Everything here is syzkaller. So I would say "Allocate what executor normally allocates", or "Allocate what we normally allocate during fuzzing".

Reply:
it is vma[24] so it is not `SYZ_NUM_PAGES * SYZ_PAGE_SIZE`. "executor" is a better word here, agree.